database/postgreSQL/schema.sql

```diff
@@ -0,0 +1,43 @@
+
+DROP TABLE IF EXISTS businesses;
```

Comment:
Wouldn't it be more intuitive to 'create table if not exists' rather than to drop the table, re-create it, then reload all the data?